nservicebus/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -59,6 +61,28 @@ Each endpoint uses its own Azure storage account, thereby increasing message thr
 
 ![Scale out with multiple storage accounts](azure03.png "width=50%")
 
+## Securing connection strings to storage accounts for Scale Out
+
+When the message throughput is increased by using multiple storage accounts, its worth to consider using their names instead of raw connection strings. When applied, raw connection string values are replaced with registered names removing the possibility of leaking a connection string value. The concept of [securing connection strings to storage accounts](/nservicebus/azure-storage-queues/configuration.md#securing-connection-strings-to-storage-accounts) has been introduced in `NServiceBus.Azure.Transports.WindowsAzureStorageQueues` Version 7. When using a single account, securing connection string is limited to calling `.UseAccountNamesInsteadOfConnectionStrings()`. When Scaling Out is applied, securing connection strings requires using logical names of the accounts.
+Consider the following example:
+
+- two endpoints using different accounts `account_A` and `account_B` for their input queues
+- the `account_A` endpoint uses account with the following connection string `account_A_connection_string`
+- the `account_B` endpoint uses account with the following connection string `account_B_connection_string`
+- every endpoint sends/replies to messages to the other using `@` notation
+ - `queue@account_A` is a `queue` of the `account_A` endpoint where `account_B` endpoint sends messages to
+ - `queue@account_B` is a `queue` of the `account_B` endpoint where `account_A` endpoint sends messages to
+
+To enable sending from `account_A` to `account_B`, following configuration has to be applied in the `account_A` endpoint
+
+snippet:AzureStorageQueueUseMultipleAccountNamesInsteadOfConnectionStrings1
+
+To enable sending from `account_B` to `account_A`, following configuration has to be applied in the `account_B` endpoint
+
+snippet:AzureStorageQueueUseMultipleAccountNamesInsteadOfConnectionStrings2
+
+`DefaultAccountName(string name)` maps a name of an account used by this endpoint.
+`AccountRouting().AddAccount(string accountName, string connectionString)` adds a mapping for other accounts, enabling using `@` notation for destination addresses `queue_name@accountName`.
 
```

Comment:
maybe add something here about the pitfall when using request/reply, that 'default account name' may imply something different to the replier than it does to the sender